domains/oxus.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "oxus",
+    "email": "oxusescu@gmail.com"
+  },
+  "record": {
+    "CNAME": "staycanuca.github.io/booksearch/"
```

cname can't have protocol or slashes
```suggestion
    "CNAME": "staycanuca.github.io"
```